test/apiv2/26-containersWait.at

```diff
@@ -0,0 +1,47 @@
+# -*- sh -*-
+#
+# test more container-related endpoints
+#
+
+red='\e[31m'
+nc='\e[0m'
+
+podman pull "${IMAGE}" &>/dev/null
+
+# Ensure clean slate
+podman rm -a -f &>/dev/null
+
+CTR="WaitTestingCtr"
+
+t POST "containers/nonExistent/wait?condition=next-exit" '' 404
+
+podman create --name "${CTR}" --entrypoint '["sleep", "0.5"]' "${IMAGE}"
```

Comment:
I wish I could have just `--entrypoint '["true"]'` here, however such a container exits too fast. :disappointed:

Reply:
meaning the container is in `running` state for such a short duration we are not capable of detecting that